praktikumsplaner-backend/src/main/java/de/muenchen/oss/praktikumsplaner/rest/PraktikumsstellenController.java

```diff
@@ -80,12 +80,17 @@ public TreeMap<String, List<PraktikumsstelleDto>> getAllPraktiumsstellenInSpecif
 
     @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
     @PatchMapping("/{praktikumsstellenId}")
-    @ResponseStatus(HttpStatus.OK)
     public PraktikumsstelleDto assignNwk(@PathVariable(name = "praktikumsstellenId") UUID praktikumsstellenId,
             @RequestParam(name = "nwkId", required = false) UUID nwkId) {
         if (nwkId == null) {
             return praktikumsstellenService.unassignNwk(praktikumsstellenId);
         }
         return praktikumsstellenService.assignNwk(praktikumsstellenId, nwkId);
     }
+
+    @PreAuthorize("hasRole('ROLE_' + T(de.muenchen.oss.praktikumsplaner.security.AuthoritiesEnum).AUSBILDUNGSLEITUNG.name())")
+    @DeleteMapping("/{praktikumsstellenId}")
+    public void deletePratkikumsstelle(@PathVariable(name = "praktikumsstellenId") UUID praktikumsstellenId) {
+        praktikumsstellenService.deletePraktikumsstelle(praktikumsstellenId);
+    }
```

Comment:
this controller has 2 endpoints for creating a praktikumsstelle. Why only one to delete them? Why not two, one for studium and one for ausbildung, too?

Reply:
Differentiation is necessary during creation due to the different structure of the objects. When deleting based on the ID, this differentiation is possible but not necessary in my view, as this requires a more complex structure for differentiating what is to be deleted.

Reply:
for me it would be a cleaner api. I agree that the frontend has to know what type of stelle it is that should be deleted.